src/main/java/org/folio/rest/impl/BaseApi.java

```diff
@@ -85,27 +85,37 @@ protected HttpException handleProcessingError(Throwable throwable) {
   }
 
   public Response buildErrorResponse(Throwable throwable) {
-    return buildErrorResponse(handleProcessingError(throwable));
+  logger.error("Exception encountered", throwable.getCause());
```

Comment:
The indentation on this line seems to be off by a few spaces.  Please correct this.

Reply:
done